src/openrct2-ui/interface/Window.cpp

```diff
@@ -1131,4 +1172,9 @@ namespace OpenRCT2::Ui::Windows
             WindowZoomOut(*mainWindow, atCursor);
     }
 
+    int16_t GetTitleBarHeight()
+    {
+        return Config::Get().interface.EnlargedUi ? 24 : 12;
```

Comment:
I think this can be inlined into `ResizeFrame`, its only user. Haven't done so in case I missed something.

Reply:
To me, this seems cleaner. May be a good idea to replace the magic numbers, though.

Reply:
Sure. I have added some constants.